### PR TITLE
feat: manage admin table and edits

### PR DESCRIPTION
**Pull request by gdcho**
https://www.loom.com/share/2d03525a00104560a9c760bb93598934?sid=999ffb35-2f85-4815-abc6-4cc62e60da38

### CODE REVIEW

**Comment by gdcho** [comment]
still need to rebase with main

**Comment by jasper-oh** [comment]
Is this draft?

**Comment by gdcho** [comment]
seems there's no merge conflict? so no need to rebase

File: card-game-client/app/components/ui/admin-card.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+import GameCard from "./game-card";
+import Image from "next/image";
+
+const categories = [
+  {
+    id: 1,
+    category: "Movies",
+    subcategory: "Harry Potter",
+    question: "Who played Harry Potter in the movies?",
+    image:
```
Comment (jasper-oh) [inline card-game-client/app/components/ui/admin-card.tsx:20]: don't know might be edit later?
 because this is just mock data

**Comment by gdcho** [comment]
still need to add card feature but will open separate pr with that along with refractor

**Comment by jasper-oh** [comment]
🍏

**Review by jasper-oh — APPROVED** [review]
🍏

File: card-game-client/app/components/ui/admin-card.tsx (hunk 1 above)
Comment (gdcho) [reply card-game-client/app/components/ui/admin-card.tsx:20]: yes, will remove while implementing api

**Pull request merged**